mongodbatlas/resource_mongodbatlas_org_invitation.go

```diff
@@ -245,3 +247,18 @@ func splitOrgInvitationImportID(id string) (orgID, username string, err error) {
 
 	return
 }
+
+func validateOrgInvitationAlreadyAccepted(ctx context.Context, conn *MongoDBClient, username, orgID string) (bool, error) {
+	user, _, err := conn.Atlas.AtlasUsers.GetByName(ctx, username)
+	if err != nil {
+		return false, err
+	}
+
+	for _, role := range user.Roles {
+		if role.OrgID == orgID {
```

Comment:
Our slack conversation dropped off but I think we should explain to the user what happened somehow. Basically the user will have no way to know how we handle when the user accepts.  We should at least document this.  That when the user accepts the invitations the provider will validate that they are now a member of the organization.  That when that happens we will remove the invitation from state and suggest the user removes the invitation from the Terraform configuration, unless they want the user immediately re-invited upon leaving.

Reply:
I added more documentation to the org_invitations: 

> **NOTE**: The provider also will validate 3 cases:
>* If the user haven't accepted the invitation, nothing happens.
>* If the user already belongs to the organization, the provider will remove the invitation from the state, but it's necessary to remove from the terraform resource definition.
>* If the user is not a member of the organization or rejected the invitation, the provider will try to send again the invitation (unless the invitation get removed from the terraform resource definition).